server/src/main/scala/za/co/absa/atum/server/api/http/Endpoints.scala

```diff
@@ -185,6 +197,19 @@ trait Endpoints extends BaseEndpoints {
       .errorOutVariantPrepend(errorInDataOneOfVariant)
   }
 
+  protected val getAncestorsEndpointV2
+  : PublicEndpoint[(Long, Option[Int], Option[Long]), ErrorResponse, PaginatedResponse[
+    PartitioningWithIdDTO
+  ], Any] = {
+    apiV2.get
+      .in(V2Paths.Partitionings / path[Long]("partitioningId") / V2Paths.Partitionings)
```

Comment:
the path is very likely incorrect ... 
according to this design https://github.com/AbsaOSS/atum-service/issues/141 it should be /partitionings/{id}/parents
potentially we can thinks of using 'ancestors' instead

Reply:
You are correct, again I think I was doing some testing. I will change it as necessary.

Reply:
> the path is very likely incorrect ... according to this design #141 it should be /partitionings/{id}/parents potentially we can thinks of using 'ancestors' instead

`ancestors` is better IMHO.

Reply:
Yeah why not, I'm okay with it. ALthough, potentially consider writing something short about it here in our Vocabulary documentation section: https://github.com/AbsaOSS/atum-service?tab=readme-ov-file#Vocabulary